snort-ip-blocklist-feed/info.json

```
@@ -7,7 +7,7 @@
   "publisher": "Fortinet",
   "cs_approved": true,
   "cs_compatible": true,
-  "help_online": "https://docs.fortinet.com/document/fortisoar/1.0.0/snort-ip-blocklist-feed/754/snort-ip-blocklist-feed-1-0-0",
+  "help_online": "",
   "icon_small_name": "small.png",
   "icon_large_name": "large.png",
   "ingestion_supported": true,
```

Please add a checkbox parameter for "Terms and Conditions" in the configuration. The connector should only function if the client has selected (accepted) it. Otherwise, it should throw an error during the health check: "Please accept the Terms and Conditions."